apple/MarkdownTextInputDecoratorView.mm

```diff
@@ -110,17 +140,8 @@ - (void)setMarkdownStyle:(RCTMarkdownStyle *)markdownStyle
   _markdownStyle = markdownStyle;
   [_markdownUtils setMarkdownStyle:markdownStyle];
 
-  if (_textView != nil) {
-    // We want to use `textStorage` for applying markdown when possible. Currently it's only available for UITextView
-    [_textView textDidChange];
-  } else {
-    // apply new styles
-#ifdef RCT_NEW_ARCH_ENABLED
-    [_textInput _setAttributedString:_backedTextInputView.attributedText];
-#else
-    [_textInput setAttributedText:_textInput.attributedText];
-#endif /* RCT_NEW_ARCH_ENABLED */
-  }
+  // trigger reformatting
+  [_textView.textStorage setAttributedString:_textView.attributedText];
```

Comment:
Shouldn't `textField` also be reformatted here?

Reply:
Good catch, thanks! Fixed in 9ff01d5.

<table>
<thead>
<tr>
<th>Before</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td><video src="https://github.com/user-attachments/assets/e650ce76-ac71-4335-b8f4-e619d03d534b" /></td>
<td><video src="https://github.com/user-attachments/assets/6d3cd93a-f903-4887-83c0-51964f0a1f59" /></td>
</tr>
</tbody>
</table>